lib/components/regions_map/interactor/usecases/get_places_from_coordinates_use_case.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:geocoding/geocoding.dart';
+import 'package:seeds/datasource/remote/geocoding_service.dart';
+import 'package:seeds/domain-shared/base_use_case.dart';
+
+class GetPlacesFromCoordinatesUseCase extends InputUseCase<List<Placemark>, _Input> {
+  static _Input input({required double lat, required double lng, String? localeIdentifier}) =>
+      _Input(lat: lat, lng: lng, localeIdentifier: localeIdentifier);
```

Comment:
@gguijarro-c-chwy  Do we really need a static method? why not just create a new _Input instance as parameter??

Reply:
try it out. Let me know how it goes :)